addons/pos_self_order/controllers/webmanifest.py

```diff
@@ -0,0 +1,38 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import mimetypes
+import re
+
+from odoo import http
+from odoo.http import request
+from odoo.addons.web.controllers.webmanifest import WebManifest as WebWebManifest
+
+
+class WebManifest(WebWebManifest):
+    def _get_scoped_app_name(self, app_id):
+        if app_id == "pos_self_order":
+            res_id = re.findall(r'pos-self/(\d+)', request.params['path'])[0]
+            record = request.env['pos.config'].search([('id', '=', res_id)])
+            if record:
+                return record.name
+        return super()._get_scoped_app_name(app_id)
+
+    def _get_scoped_app_icons(self, app_id):
+        if app_id == "pos_self_order":
+            company = request.env.company
+            if company.uses_default_logo:
+                icon_src = '/point_of_sale/static/description/icon.svg'
+            else:
+                icon_src = '/web/image?model=res.company&id=%s&field=logo&crop=true&height=%s&width=%s' % (company.id, 192, 192)
```

Comment:
%-string are very much python2, also the height and width are hardcoded
```suggestion
                icon_src = f'/web/image?model=res.company&id={company.id}&field=logo&crop=true&height=192&width=192'
```

Reply:
or, if you thing it deserves it
```py
from urllib.parse import urlencode
```
```suggestion
                icon_src = '/web/image?' + urlencode({
                    'model': 'res.company',
                    'id': company.id,
                    'field': 'logo',
                    'crop': True,
                    'height': 192,
                    'width': 192,
                })
```

Reply:
why croppping btw?

Reply:
> why croppping btw?

You're right, it makes no sense to crop